cardano_node_tests/tests/test_plutus_mint_build.py

```diff
@@ -625,10 +625,16 @@ def test_witness_redeemer(
             mint=plutus_mint_data,
             required_signers=[signing_key_golden],
         )
+        # sign incrementally (just to check that it works)
         tx_signed_step2 = cluster.sign_tx(
             tx_body_file=tx_output_step2.out_file,
-            signing_key_files=tx_files_step2.signing_key_files,
-            tx_name=f"{temp_template}_step2",
+            signing_key_files=[issuer_addr.skey_file],
+            tx_name=f"{temp_template}_step2_sign0",
+        )
+        tx_signed_step2 = cluster.sign_tx(
```

Comment:
**optional** maybe we could give different variable names? e.g. tx_signed_step2 and tx_signed_step2_incremented, just to be easy to differentiate between them